project/Dependencies.scala

```diff
@@ -13,61 +13,57 @@
 import sbt._
 
 object Dependencies {
-
-  val resolutionRepos = Seq(
+  val resolvers = Seq(
     "Snowplow Analytics Maven releases repo" at "http://maven.snplow.com/releases/",
-    "Snowplow Analytics Maven snapshot repo" at "http://maven.snplow.com/snapshots/",
-    "Twitter maven repo"                     at "http://maven.twttr.com/",
-    // For Scalazon
-    "BintrayJCenter"                         at "http://jcenter.bintray.com"
+    "Twitter maven repo"                     at "http://maven.twttr.com/"
   )
 
   object V {
     // Java
-    val slf4j                = "1.7.6"
-    val kinesisClient        = "1.4.0"
-    val kinesisConnector     = "1.1.2"
-    val hadoop               = "1.2.1"
-    val elephantbird         = "4.5"
-    val yodaTime             = "2.2"
-    val yodaConvert          = "1.2"
+    val slf4j            = "1.7.6"
+    val kinesisClient    = "1.7.5"
+    val kinesisConnector = "1.3.0"
+    val hadoop           = "2.7.3"
+    val elephantbird     = "4.15"
+    val hadoopLZO        = "0.4.20"
+    val yodaTime         = "2.9.9"
```

Comment:
This should be renamed to `jodaTime` not `yodaTime` ... not sure why this was ever spelled this way.